### DISCUSSION TITLE
Laravel/Jetstream Tailwind output.css

### DISCUSSION THREAD

**Discussion by DylanDec** [laravel/framework#34624] [Q&A] [answered]
Hello

I want to use auto completion in my IDE (phpStorm).

How can I get Tailwind into output.css. I can't find where tailwind.css is located.

Thanks in advance

**Comment by AlexWinder** [comment, marked as answer]
Check your webpack configuration, it will tell you how things are compiled. Tailwind is obtained from a `tailwind.config.js` file and then you import it into your `app.scss` or equivalent file.

https://tailwindcss.com/docs/installation
